.circleci/config.yml

```diff
@@ -215,6 +215,67 @@ jobs:
       # - store_test_results:
       #     path: ./test/results
 
+  test-integration-pm4ml:
+    executor: default-machine
+    environment: *defaults_environment
+    steps:
+      - checkout
+      - run:
+          <<: *defaults_configure_nvm
+      - run:
+          name: Install Yarn dependencies
+          command: yarn install --immutable
+      - attach_workspace:
+          at: /tmp
+      - run:
+          name: Load the pre-built docker image from workspace
+          command: docker load -i /tmp/docker-image.tar
+      ## This shouldn't be needed as the docker image is pre-built by build-local job
+      # - run:
+      #     name: Build docker-compose
+      #     command: |
+      #       docker-compose build
+      - run:
+          name: Generate certificates
+          command: |
+            cd docker/haproxy/tls
+            sh createSecrets.sh
+            cd ../../../
```

Comment:
```suggestion
            cd $CIRCLE_WORKING_DIRECTORY
```

Reply:
Thanks, Not committing this here and I handled this manually to individual extra CI job runs.